src/auth/shibboleth.js

```diff
@@ -3,15 +3,14 @@ const { createOrUpdateUser, addJwtCookie, isSafeUrl } = require('./util')
 const safeAsync = require('../middleware/safeAsync')
 
 module.exports = safeAsync(async (req, res) => {
-  // Get the user's NetID based on the "eppn" header
-  const email = req.get('eppn') || ''
-  if (email.indexOf('@') === -1) {
+  // Get the user's email based on the "eppn" header
+  const uid = req.get('eppn') || ''
```

Comment:
@nwalters512  For broader compatibility, can we change this to:
  const email = req.get('eppn') || req.get('edupersonprincipalname') || ''
(UBC uses the full label...)